Changelog.md

```diff
@@ -9,6 +9,7 @@ Compiler Features:
 Bugfixes:
  * Code Generator: Fix a crash when using ``@use-src`` and compiling from Yul to ewasm.
  * SMTChecker: Fix internal error when an unsafe target is solved more than once and the counterexample messages are different.
+ * Fix Invalid IR generation error when copying dynamically sized storage arrays
```

Comment:
```suggestion
 * IR Generator: Fix IR syntax error when copying storage arrays of functions.
```
(want to make clear that it was not "wrong IR" but "IR that does not compile", which is a huge difference)